file/provider/create_volume.go

```diff
@@ -135,44 +152,45 @@ func (vpcs *VPCSession) CreateVolume(volumeRequest provider.Volume) (volumeRespo
 }
 
 // validateVolumeRequest validating volume request
-func validateVolumeRequest(volumeRequest provider.Volume) (models.ResourceGroup, int64, error) {
+func validateVolumeRequest(volumeRequest provider.Volume) (models.ResourceGroup, int64, int32, error) {
 	resourceGroup := models.ResourceGroup{}
 	var iops int64
 	iops = 0
+	var bandwidth int32
+	bandwidth = 0
 
 	// Volume name should not be empty
-	if volumeRequest.Name == nil {
-		return resourceGroup, iops, userError.GetUserError("InvalidVolumeName", nil, nil)
-	} else if len(*volumeRequest.Name) == 0 {
-		return resourceGroup, iops, userError.GetUserError("InvalidVolumeName", nil, *volumeRequest.Name)
+	if volumeRequest.Name == nil || len(*volumeRequest.Name) == 0 {
+		return resourceGroup, iops, bandwidth, userError.GetUserError("InvalidVolumeName", nil, nil)
 	}
 
-	// Capacity should not be empty
 	if volumeRequest.Capacity == nil {
-		return resourceGroup, iops, userError.GetUserError("VolumeCapacityInvalid", nil, nil)
+		return resourceGroup, iops, bandwidth, userError.GetUserError("VolumeCapacityInvalid", nil, nil)
 	} else if *volumeRequest.Capacity < minSize {
```

Comment:
with this change user will not be able to create share with less then 10 GB as RFS based share capacity can be less then 10 as well